### PR TITLE
blog(embertimes97): emberfest cfp

### PR DESCRIPTION
**Pull request by chrisrng**

### CODE REVIEW

File: source/2019-05-04-the-ember-times-issue-97.html [hunk 1]
```diff
@@ -35,12 +36,13 @@
...
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+## [EmberFest 2019 CFP is now open! 📝](https://cfp.emberfest.eu)
+
+[EmberFest](https://emberfest.eu/) – the European Community Ember Conference – will be held at [Mogens Dahl](https://mogensdahl.dk) in Copenhagen this year from October 17-18, 2019. They have recently [announced](https://twitter.com/EmberFest/status/1125441719921717251) that the CFP is **now open** with a deadline at August 1, 2019.
+
+EmberFest is looking for a wide range of talks including in-depth technical talks along with broader talks covering other aspects of software development. While Ember-related is best but we're open to considering talks that cover more general software / tech related topics that would be of interest to Ember developers.
```
Comment (ijlee2) [inline source/2019-05-04-the-ember-times-issue-97.html]:
```suggestion
EmberFest is looking for a wide range of talks, from in-depth technical talks to broader talks covering other aspects of software development. While Ember-related is best, EmberFest is open to considering talks that cover more general software / tech related topics that would be of interest to Ember developers.
```

File: source/2019-05-04-the-ember-times-issue-97.html [hunk 2]
```diff
@@ -13,6 +13,7 @@
 This week:
 ...
 interview with **Yehuda Katz** 🎙️,
+the EmberFest 2019 CFP is now open 📝,
```
Comment (ijlee2) [inline source/2019-05-04-the-ember-times-issue-97.html:15]: I think the CFP has more importance than the interview with Yehuda (which happened a month ago). Do you want to place your announcement on top?

**Review by ijlee2 — APPROVED** [review]
Looks great. Thank you!

File: source/2019-05-04-the-ember-times-issue-97.html (hunk 2 above)
Comment (chrisrng) [reply source/2019-05-04-the-ember-times-issue-97.html:15]: Ya lets do it

**Pull request merged**